src/Microsoft.AspNetCore.Razor/Editor/SpanEditHandler.cs

```diff
@@ -29,8 +29,9 @@ public SpanEditHandler(Func<string, IEnumerable<ISymbol>> tokenizer, AcceptedCha
         public AcceptedCharacters AcceptedCharacters { get; set; }
 
         /// <summary>
-        /// Provides a set of hints to editors which may be manipulating the document in which this span is located.
+        /// This property is obsolete and will be removed in a future version.
```

Comment:
Didn't feel like any extra information would be beneficial since this property is not currently set by the Razor system. This comment transitively applies to the above `EditorHints` class.

If you feel differently I can add a little snippet stating that it's not set by Razor.